integration_tests/conftest.py

```diff
@@ -34,14 +34,6 @@
 TRINO_PORT = 8080
 
 
-def is_process_alive(pid):
```

Comment:
It's surprising that this is unused. I'll check what changed to make this unused.

Reply:
This function was never called but has been here since the beginning. 🤷‍♂️ 

Let's remove it then. 👍